pos_invoice_pay/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "summary": """Handle the payment process for Sale Orders/Invoices over Point of Sale""",
     "category": "Point of Sale",
     "images": ["images/pos_invoice_pay_main.png"],
-    "version": "10.0.1.0.6",
+    "version": "10.0.1.1.0",
```

Comment:
Лучше капитально поменять версию, т.к. там поля добавляются и нужно нажимать "Обновить модуль"
```suggestion
    "version": "10.0.2.0.0",
```

Reply:
> там поля добавляются и нужно нажимать "Обновить модуль"

https://gitlab.com/itpp/handbook/-/blob/master/technical-docs/__manifest__.md#version

> The x.y.z version numbers follow the semantics breaking.feature.fix:

> - x increments when the data model or the views had significant
changes. Data migration might be needed, or depending modules might
be affected.

> - y increments when non-breaking new features are added. A module
upgrade will probably be needed.

Попадает под `y`.